draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2441,6 +2441,11 @@ Host: www.example.org
    first verifying that the intercepted connection is targeting a
    valid IP address for that host.
 </t>
+<t>
+   Note that some versions of HTTP define alternative means of transferring the
+   host and port information of the target URI; see the :authority psuedo-header
```

Comment:
Typo: pseudo-header

Reply:
Thank you for noticing that. Fixed in 0ad7c9cc.